guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -0,0 +1,122 @@
+[id="configuring-the-active-directory-authentication-source-on-projectserver_{context}"]
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Prerequisites
+* The base system of your {ProjectServer} must be joined to an Active Directory (AD) domain.
+To enable AD users to sign in with Kerberos single sign-on, join the system by using the System Security Services Daemon (SSSD) and Samba services.
++
+Install the following packages on {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {project-package-install} krb5-workstation oddjob-mkhomedir oddjob realmd samba-winbind-clients samba-winbind samba-common-tools samba-winbind-krb5-locator sssd
+----
++
+Specify the required software when joining the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# realm join _AD.EXAMPLE.COM_ --membership-software=samba --client-software=sssd
+----
++
+For more information on direct AD integration, see link:{RHELDocsBaseURL}9/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#connecting-rhel-systems-directly-to-ad-using-samba-winbind_integrating-rhel-systems-directly-with-active-directory[Connecting RHEL systems directly to AD using Samba Winbind].
+
+.Procedure
+. Define AD realm configuration in a location where {foreman-installer} expects it:
+.. Create a directory named `/etc/ipa/`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# mkdir /etc/ipa
+----
++
+.. Create a file named `default.conf` in the `/etc/ipa/` directory:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# touch /etc/ipa/default.conf
+----
+.. Add the following lines to `/etc/ipa/default.conf` to configure the Kerberos realm for the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[global]
+realm = _AD.EXAMPLE.COM_
+----
+. Configure the Apache keytab for Kerberos connections:
+.. Create a file named `/etc/net-keytab.conf` to store configuration details for Samba:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# touch /etc/net-keytab.conf
+----
++
+.. Add the following lines to `/etc/net-keytab.conf` to provide Samba with details for how to interact with AD:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[global]
+workgroup = _AD.EXAMPLE_
+realm = _AD.EXAMPLE.COM_
+kerberos method = system keytab
+security = ads
+----
++
+.. Add the Kerberos service principal to the keytab file at `/etc/httpd/conf/http.keytab`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# KRB5_KTNAME=FILE:/etc/httpd/conf/http.keytab net ads keytab add HTTP -U Administrator -s /etc/net-keytab.conf -d3
```

Comment:
Is debug level 3 (-d3) intentional?

Reply:
It is. There is no output if creating the keytab fails and it took me a while to figure out what was happening. Level 3 debugging was used in the legacy procedure, perhaps a different level would suffice..? I'll try to tweak this when I re-test the procedure and update if necessary.

Reply:
I checked the [man pages](https://www.samba.org/samba/docs/current/man-html/net.8.html) and anything higher than 1 is indeed supposed to be used for troubleshooting. So I dropped it after all.